recipe_scrapers/_schemaorg.py

```diff
@@ -132,7 +132,7 @@ def _read_duration_field(self, k: str) -> int | None:
             return None
         if isinstance(v, str):
             return get_minutes(v)
-        # Workaround: strictly speaking schema.org does not provide for minValue and maxValue properties on objects of type Duration; they are however present on objects with type QuantitativeValue
+        # Workaround: Strictly speaking schema.org does not provide for minValue and maxValue properties on objects of type Duration; they are however present on objects with type QuantitativeValue
```

Comment:
Not sure about this.  a `TODO: ` entry is like a marker that occurs before a sentence (so a capital letter is good after it), but in the context here, `Workaround: strictly` is OK I think - `strictly` is a continuation of the sentence, containing an explanation of the workaround.

```suggestion
        # Workaround: strictly speaking schema.org does not provide for minValue and maxValue properties on objects of type Duration; they are however present on objects with type QuantitativeValue
```